searchcore/src/tests/proton/matching/matching_stats_test.cpp

```diff
@@ -184,11 +193,15 @@ TEST("requireThatPartitionsAreAddedCorrectly") {
     EXPECT_EQUAL(0.5, all1.getPartition(0).wait_time_min());
     EXPECT_EQUAL(1.0, all1.getPartition(0).active_time_max());
     EXPECT_EQUAL(0.5, all1.getPartition(0).wait_time_max());
+    EXPECT_EQUAL(1u, all1.getPartition(0).softDoomed());
+    EXPECT_EQUAL(1000, all1.getPartition(0).doomOvertime());
 
     MatchingStats::Partition otherSubPart;
     otherSubPart.docsCovered(7).docsMatched(3).docsRanked(2).docsReRanked(1)
-        .active_time(0.5).wait_time(1.0);
+            .active_time(0.5).wait_time(1.0).softDoomed(true).doomOvertime(-300);
```

Comment:
dooming with negative overtime seems improbable and maybe a bad example?

Reply:
Fixed